packages/ses/README.md

```diff
@@ -1,7 +1,17 @@
-# Secure ECMAScript (SES)
+# SES
 
-Secure ECMAScript (SES) is an execution environment that provides fine-grained
-sandboxing with Compartments.
+SES is a slight modification to the JavaScript language that enables programs
+to defend their own correctness, mostly by disabling dangerous features.
+JavaScript without SES is highly reconfigurable at runtime, so programs can
+easily interfere with each other by altering shared objects.
```

Comment:
"highly reconfigurable" sounds attractive, but it's unattractive.

As far as I can tell, "interfere with each other" used in this way is jargon that only people following Mark would know. 

```suggestion
Running JavaScript without SES leaves your program vulnerable to any other code that executes in the same environment, by altering shared objects.
```